src/handlers/farmerHandlers.js

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import { pathObj } from '../utils/generalVariables';
+
+export const getFarmersHandler = (token) => {
```

Comment:
I wouldn't pass the token as prop here and just read it from local storage inside the handler. One user shouldn't have access to any other token anyway

Reply:
We want these handlers to be as self-contained as possible, so I opted to leave the token retrieval to the component.